k8s/supportability/src/collect/common.rs

```diff
@@ -23,6 +23,8 @@ pub(crate) struct DumpConfig {
     /// Topologer implements functionality to build topological infotmation of system
     pub(crate) topologer: Option<Box<dyn Topologer>>,
     pub(crate) output_format: OutputFormat,
+    /// Collect current pod logs along with historical logs
+    pub(crate) k8s_logs_override: bool,
```

Comment:
The word "override" semantically implies we are over-riding some functionality. However, here it seems we are only extending the functionality.

Reply:
By override here I meant change the default behaviour, added in that sense. The default behaviour is to not collect this log.